docker-compose.yml

```diff
@@ -47,8 +47,8 @@ services:
             - MENDER_ARTIFACT_VERSION
             - MENDER_VERSION
             - MENDER_DEB_PACKAGE_VERSION
-            - HAVE_DEVICECONNECT
-            - HAVE_DEVICECONFIG
+            - HAVE_DEVICECONNECT=1
+            - HAVE_DEVICECONFIG=1
```

Comment:
maybe we should assert for this to be on, either in the docker compose manager, or through it in the tests.

Reply:
👌 we do in the gui e2e tests - at least for device connect... we could add a trigger for changes to the compose files that causes the gui tests to run, but I think the gui tests run frequently enough that we catch these early enough 😄